synx.gemspec

```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rspec", "~> 2.14"
   spec.add_development_dependency "pry", "~> 0.9"
 
-  spec.add_dependency "clamp", "~> 0.6"
-  spec.add_dependency "colorize", "~> 0.7"
-  spec.add_dependency "xcodeproj", "~> 0.28.2"
+  spec.add_dependency "clamp", ">= 0.6"
+  spec.add_dependency "colorize", ">= 0.7"
+  spec.add_dependency "xcodeproj", ">= 0.28.2"
```

Comment:
The CocoaPods team are working on a stable 1.0 release. When this is out this could also be `~> 1.0`.